### PR TITLE
Jgpuga/mlops

### PR DESCRIPTION
**Pull request by javiergp**
New BQ ML pipeline for Vertex MLOPs enterprise

### CODE REVIEW

File: examples/vertex_mlops_enterprise/.github/workflows/deploy.yml.TEMPLATE [hunk 1]
```diff
@@ -48,5 +48,5 @@ jobs:
         access_token_lifetime: 3600s
 
     - name: 'Deploy model'
```
Comment (svidiella) [inline examples/vertex_mlops_enterprise/.github/workflows/deploy.yml.TEMPLATE:50]:
```suggestion
    - name: 'Deploy TFX model'
```

File: examples/vertex_mlops_enterprise/.github/workflows/deploy.yml.TEMPLATE [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Deploy tfx model
+name: Deploy Vertex AI tfx model
```
Comment (svidiella) [inline examples/vertex_mlops_enterprise/.github/workflows/deploy.yml.TEMPLATE:15]: @javiergp should we change the name to deploy-tfx.yml.TEMPLATE?

File: examples/vertex_mlops_enterprise/.gitignore [hunk 3]
```diff
@@ -4,3 +4,4 @@ terraform.tfstate*
 .DS_Store
 **/__pycache__/**
 venv
+**/.ipynb_checkpoints/**
```
Comment (svidiella) [inline examples/vertex_mlops_enterprise/.gitignore:7]:
```suggestion
**/.ipynb_checkpoints/**

```

File: examples/vertex_mlops_enterprise/doc/01-ENVIRONMENTS.md [hunk 4]
```diff
@@ -45,8 +45,8 @@ Setup your new Github repo using the Github web console or CLI.
 Copy the `vertex_mlops_enterprise` folder to your local folder, including the Github actions:
 
 ```
-cp -R ./examples/vertex_mlops_enterprise/* ./<YOUR LOCAL FOLDER>
-cp -R ./examples/vertex_mlops_enterprise/.github ./<YOUR LOCAL FOLDER>
+# Copy base directory from git repo, including hidden dirs and files
```
Comment (svidiella) [inline examples/vertex_mlops_enterprise/doc/01-ENVIRONMENTS.md:48]: @javiergp it's a bit redundant with the line 45

File: examples/vertex_mlops_enterprise/.github/workflows/deploy.yml.TEMPLATE (hunk 2 above)
Comment (javiergp) [reply examples/vertex_mlops_enterprise/.github/workflows/deploy.yml.TEMPLATE:15]: I'll add this in the TODO list

File: examples/vertex_mlops_enterprise/src/bqml_pipeline/src/pipeline.py [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+#            name="accuracy is above threshold",
+#        ):
+
+#            endpoint_create_op = EndpointCreateOp(
+#                project=project, location=region, display_name=ENDPOINT_NAME
+#            )
+
+#            _ = upload_model_enpoint(
+#                project=project, location=region, bq_model_name=model
+#            ).after(endpoint_create_op)
```
Comment (svidiella) [inline examples/vertex_mlops_enterprise/src/bqml_pipeline/src/pipeline.py:182]: @javiergp is this something we want to keep?

**Review by svidiella — APPROVED** [review]
Some NITs, LGTM!

File: examples/vertex_mlops_enterprise/doc/01-ENVIRONMENTS.md (hunk 4 above)
Comment (javiergp) [reply examples/vertex_mlops_enterprise/doc/01-ENVIRONMENTS.md]: fixed

File: examples/vertex_mlops_enterprise/src/bqml_pipeline/src/pipeline.py (hunk 5 above)
Comment (javiergp) [reply examples/vertex_mlops_enterprise/src/bqml_pipeline/src/pipeline.py]: removed

**Pull request merged**